File: frontend/src/services/bookingService.js
import axios from "axios";

const API = "http://127.0.0.1:8000";

export const createBooking = async (data) => {
  const response = await axios.post(
    `${API}/bookings`,
    data
  );

  return response.data;
};

export const getBookings = async () => {
  const response = await axios.get(
    `${API}/bookings`
  );

  return response.data;
};

File: backend/services/booking_service.py
from pymongo import MongoClient

from config import MONGO_URL, DB_NAME

client = MongoClient(MONGO_URL)

db = client[DB_NAME]

collection = db["bookings"]


def create_booking(data):
    result = collection.insert_one(data)

    data["_id"] = str(result.inserted_id)

    return data


def get_bookings():
    bookings = list(collection.find())

    for booking in bookings:
        booking["_id"] = str(booking["_id"])

    return bookings